features/cloud.py

```diff
@@ -481,6 +496,97 @@ def _create_instance(
         return inst
 
 
+class GCE(Cloud):
```

Comment:
Let's replace GCE -> GCP throughout  and gce -> gcp. as that's the official abbrevation for Google Cloud Platform